### PR TITLE
File added : assignment 5 and 6 completed

### PR DESCRIPTION
**Pull request by dayanandviveki**
review my PR

### CODE REVIEW

**Review by sourabh-kanwade — CHANGES_REQUESTED** [review]
Fix Spelling mistakes in Pull Request title

File: Dayanand Viveki/HTML/Assignment5/Biodata.html [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+    <body>
+        <div class="main-block">
+            <p class="name">Dayanand Viveki</p>
+
+            <div class="aboutme">
+                <span class="headline">
+                    <p>About Me</p>
+                </span>
+
+                <span class="info">
```
Comment (kalyani1942) [inline Dayanand Viveki/HTML/Assignment5/Biodata.html:17]: Remove Space

**Review by kalyani1942 — DISMISSED** [review]

File: Dayanand Viveki/HTML/Assignment5/Biodata.html (hunk 1 above)
Comment (dayanandviveki) [reply Dayanand Viveki/HTML/Assignment5/Biodata.html:17]: changes done

Comment (dayanandviveki) [reply Dayanand Viveki/HTML/Assignment5/Biodata.html:17]: changes done

**Review by sourabh-kanwade — APPROVED** [review]

**Review by Pawan1005 — CHANGES_REQUESTED** [review]
add new line at the end of CSS files

**Review by Prashant698 — APPROVED** [review]

**Review by somesh6797 — APPROVED** [review]

**Review by kalyani1942 — APPROVED** [review]

File: Dayanand Viveki/HTML/Assignment6/assets/css/style.css [hunk 2]
```diff
@@ -0,0 +1,44 @@
+*{
+    margin: 0px;
+    padding: 0px;
+    display: flexbox;
+}
+.font-size-35{
+    font-size: 35px;
+}
+
```
Comment (somesh6797) [inline Dayanand Viveki/HTML/Assignment6/assets/css/style.css:9]: remove this space

**Review by somesh6797 — CHANGES_REQUESTED** [review]

File: Dayanand Viveki/HTML/Assignment5/assets/css/style.css [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+    text-decoration:none;
+    font-size: 30px;
+    margin: 10px;
+    color: wheat;
+}
+.info,
+.links{
+    margin: 1rem;
+    text-align: justify;
+}
```
Comment (Pawan1005) [inline Dayanand Viveki/HTML/Assignment5/assets/css/style.css:63]: add one space before {

File: Dayanand Viveki/HTML/Assignment6/assets/css/style.css [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+    margin-left: 24px;
+    font-weight: bold;
+}
+body .form-item input{
+    font-size: 20px;
+}
+body .form-item label,
+ body .form-item input{
+    margin: 9px;
+}
```
Comment (Pawan1005) [inline Dayanand Viveki/HTML/Assignment6/assets/css/style.css:44]: add one space before {

**Review by Pawan1005 — CHANGES_REQUESTED** [review]
add one space before {

File: Dayanand Viveki/HTML/Assignment5/assets/css/style.css (hunk 3 above)
Comment (dayanandviveki) [reply Dayanand Viveki/HTML/Assignment5/assets/css/style.css:63]: ok sir

**Pull request closed without merging**